moveit_ros/planning_interface/test/launch/move_group_ompl_constraints.test.py

```diff
@@ -11,11 +11,11 @@ def generate_test_description():
 
 class TestGTestProcessActive(unittest.TestCase):
 
-    def test_gtest_run_complete(self, proc_info, ompl_constraint_test, run_move_group_node, static_tf, robot_state_publisher, fake_joint_driver_node, mongodb_server_node):
+    def test_gtest_run_complete(self, proc_info, ompl_constraint_test, run_move_group_node, static_tf, robot_state_publisher, ros2_control_node, mongodb_server_node):
         proc_info.assertWaitForShutdown(ompl_constraint_test, timeout=4000.0)
 
 @launch_testing.post_shutdown_test()
 class TestGTestProcessPostShutdown(unittest.TestCase):
 
-    def test_gtest_pass(self, proc_info, ompl_constraint_test, run_move_group_node, static_tf, robot_state_publisher, fake_joint_driver_node, mongodb_server_node):
+    def test_gtest_pass(self, proc_info, ompl_constraint_test, run_move_group_node, static_tf, robot_state_publisher, ros2_control_node, mongodb_server_node):
         launch_testing.asserts.assertExitCodes(proc_info, process=ompl_constraint_test)
```

Comment:
An empty line at the end of the file....

```suggestion
        launch_testing.asserts.assertExitCodes(proc_info, process=ompl_constraint_test)

```